docs/modeling/fitting.rst

```diff
@@ -163,3 +163,31 @@ bounds internally.
     ['fixed', 'tied', 'bounds']
     >>> fitting.SLSQPLSQFitter.supported_constraints
     ['bounds', 'eqcons', 'ineqcons', 'fixed', 'tied']
+
+Plugin Fitters
+--------------
+
+
+Fitters defined outside of astropy's core can be inserted into the
+`astropy.modeling.fitting` namespace through the use of entry points. 
+Entry points are references to importable objects a tutorial on 
+defining entry points can be found in `setuptools' documentation 
+<http://setuptools.readthedocs.io/en/latest/setuptools.html#dynamic-discovery-of-services-and-plugins>`_
+Plugin fitters are required to extend from the `~astropy.modeling.fitting.Fitter` 
```

Comment:
Change "are required" to "must".  The "are required" wording is ambiguous and can mean that one must use plugin fitters in order to extend the fitter class.